bodhi/server/models.py

```diff
@@ -626,14 +626,17 @@ def get_pkg_pushers(self, branch, settings):
 
     def get_pkg_committers_from_pagure(self):
         """
-        Pull users who can commit and are watching a package in Pagure.
+        Pull users and groups who can commit on a package in Pagure.
 
-        Return two two-tuples of lists:
-        * The first tuple is for usernames.  The second tuple is for groups.
-        * The first list of the tuple is for committers. The second is for
-          watchers.
+        Returns a tuple with two lists:
+        * The first list is for usernames that have commit access. This list
+        will contain members from Pagure groups if `pagure_uses_fas_groups` is
+        set to True.
+        * The second list is for FAS groups that have commit access. This will
+        be empty if `pagure_uses_fas_groups` is set to False.
         """
         pagure_url = config.get('pagure_url')
+        pagure_uses_fas_groups = config.get('pagure_uses_fas_groups')
```

Comment:
@bowlofeggs the setting itself is configurable in Pagure, but I can remove the setting and just assume they are FAS groups.